bigbluebutton-html5/imports/startup/server/logger.js

```diff
@@ -1,15 +1,30 @@
 import { Meteor } from 'meteor/meteor';
 import Winston from 'winston';
 
-let Logger = new Winston.Logger();
-
-// Write logs to console
-Logger.add(Winston.transports.Console, {
-  prettyPrint: false,
-  humanReadableUnhandledException: true,
-  colorize: true,
-  handleExceptions: true,
-});
+let Logger = new (Winston.Logger)({
+    transports: [
+
+      // Write logs to console
+      new Winston.transports.Console({
+            prettyPrint: false,
+            humanReadableUnhandledException: true,
+            colorize: true,
+            handleExceptions: true,
+          }),
+      ],
+    levels: { error: 0, warn: 1, info: 2, verbose: 3, debug: 4, silly: 5, },
```

Comment:
Since those are the same as the default values, do we need to explicit set them?